src/js/store/tasks/reducer.ts

```diff
@@ -156,6 +156,11 @@ const reducer = (
             },
           },
         };
+      } else if (objectType === OBJECT_TYPES.TASK && assigned_to_me) {
+        return {
+          tasks: response,
+          fetched: true,
+        };
```

Comment:
This won't work quite right, because it's expecting the returned object here to be a `TaskState` object, which has keys as product IDs, and values of `TaskByProjectState` objects. So like I said in Slack, I'd lean toward not using `fetchObjects` to get this list, and not using the reducer at all or storing these in the Redux store.